common.sh

```diff
@@ -5,12 +5,12 @@
 # Don't allow to call these scripts from their directories.
 [ -e $(basename $0) ] && echo "PLEASE USE THIS SCRIPT FROM ANOTHER DIR" && exit 1
 
-# Ensure that fuzzing engine, if defined, is either "libfuzzer", "afl", or "coverage"
+# Ensure that fuzzing engine, if defined, is valid
 FUZZING_ENGINE=${FUZZING_ENGINE:-"libfuzzer"}
-
-[[ $FUZZING_ENGINE != "libfuzzer" ]] && [[ $FUZZING_ENGINE != "afl" ]] &&\
-  [[ $FUZZING_ENGINE != "coverage" ]] && echo "USAGE: If defined, FUZZING_ENGINE\
-  should be either 'afl', 'libfuzzer', or 'coverage', but it is $FUZZING_ENGINE" && exit 1
+POSSIBLE_FUZZING_ENGINE="libfuzzer afl coverage fsanitize_fuzzer"
+!(echo "$POSSIBLE_FUZZING_ENGINE" | grep -w "$FUZZING_ENGINE" > /dev/null) && \
+  echo "USAGE: Error: If defined, FUZZING_ENGINE should be among $POSSIBLE_FUZZING_ENGINE;\
+  on the contrary, it was defined as $FUZZING_ENGINE" && exit 1
```

Comment:
I feel like it doesn't quite transfer because there is no "or", or commas, in `POSSIBLE_FUZZING_ENGINE`